src/core/transaction.rs

```diff
@@ -349,6 +845,32 @@ pub trait TxOutExt: AsPtr<btck_TransactionOutput> {
 ///
 /// Transaction outputs can be created from a script pubkey and amount, or retrieved
 /// from existing transactions. They represent spendable coins in the UTXO set.
+///
+/// # Structure
+///
+/// Each output contains:
+/// - **Value**: Amount in satoshis (1 BTC = 100,000,000 satoshis)
+/// - **Script pubkey**: The locking script that defines how the output can be spent
```

Comment:
I think we can skip this section. Should be clear what it is  from the description preceding it.